shared/params.go

```diff
@@ -373,9 +374,9 @@ func ParseMaxCountParamWithDefault(r *http.Request, defaultValue int) (count int
 	return defaultValue, nil
 }
 
-// ParseFromParam parses the "from" param as a timestamp.
-func ParseFromParam(r *http.Request) (*time.Time, error) {
-	if fromParam := r.URL.Query().Get("from"); fromParam != "" {
+// ParseDTParam parses the date/time param named "name" as a timestamp.
+func ParseDTParam(r *http.Request, name string) (*time.Time, error) {
```

Comment:
Personally I'd prefer `ParseDateTimeParam`, unless `DT` is a well established abbreviation in Go.